src/Modules/Settings/Dnn.PersonaBar.Extensions/Services/ExtensionsController.cs

```diff
@@ -982,8 +983,8 @@ public HttpResponseMessage GetModuleFiles(string ownerFolder, string moduleFolde
                         files.AddRange(GetFiles(Globals.HostMapPath + "Templates\\", ".module.template"));
                         break;
                     case FileType.Manifest:
-                        files.AddRange(GetFiles(folder, "*.dnn"));
-                        files.AddRange(GetFiles(folder, "*.dnn5"));
+                        var regex = new Regex(@"\.dnn(\d+)?$");
+                        files.AddRange(GetFiles(folder, "*.*").Where(f => regex.IsMatch(f)));
```

Comment:
I later realised that `?` will also allow non-digit character. To be 100% correct we could combine both to avoid iterating unnecessary files (is it really necessary) and the only accepting `dnn\d*`. The reg exp can actually be simplified too

```c#
var regex = new Regex(@"dnn\d*$");
files.AddRange(GetFiles(folder, "*.dnn*").Where(file => regex.IsMatch(file)));
```

Simplest possible regex is to check for optional numeric suffix once only `.dnn` file have been selected initially.

Am I overthinking this @dnnsoftware/tag? :)